dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java

```diff
@@ -251,7 +251,7 @@ public static RoutingRule insertConditionRule(RoutingRule existRule, ConditionRo
         List<String> conditions = new ArrayList<>();
         if (whiteList != null && whiteList.size() > 0) {
             StringBuilder sb = new StringBuilder();
-            sb.append("host != ");
+            sb.append("host = ");
```

Comment:
I tested this feature and it is normal. This is the document https://dubbo.apache.org/zh/docs/v2.7/user/examples/routing-rule/ , black and white uses `=>` the previous config to filter consumer.

Reply:
Thanks for your review.
When I configure, the whitelist is `1.1.1.1 `and the blacklist is `2.2.2.2`. The following screenshot is generated. According to my understanding, shouldn't it be like this? `host=1.1.1.1=> and host!=2.2.2.2 =>`, But what is shown below is `host!=1.1.1.1=> and host=2.2.2.2 =>`


![image](https://user-images.githubusercontent.com/43363120/118909519-bb42ad80-b955-11eb-8368-a5b04be4cab9.png)

![image](https://user-images.githubusercontent.com/43363120/118909525-bda50780-b955-11eb-882a-aade5cee37b0.png)

Reply:
This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
![image](https://user-images.githubusercontent.com/43994656/118910948-ecbc7880-b957-11eb-958f-1564ba48414b.png)

Reply:
> This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
> ![image](https://user-images.githubusercontent.com/43994656/118910948-ecbc7880-b957-11eb-958f-1564ba48414b.png)

Looking at my screenshot, I understand the black and white list of `consumers (not providers)` configured here, right? I configured the consumer whitelist to be 1.1.1.1, but the generated host!=1.1.1.1, isn't the allowed 1.1.1.1 filtered out?

Reply:
> This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
> ![image](https://user-images.githubusercontent.com/43994656/118910948-ecbc7880-b957-11eb-958f-1564ba48414b.png)

Thank you very much, according to your description, I have understood